WTWSMS/common/scripted_triggers/WtWSMS_scripted_triggers.txt

```diff
@@ -28,6 +28,9 @@ has_romance_culture_trigger = {
 		culture = helleno_thracian
 		culture = massalian
 		culture = romano_dacian
+		culture = spartan
+		culture = macedonian
+		culture_group = byzantine
```

Comment:
I took out Macedonians and Spartans, as you requested. It's still absurd, though.